.travis.yml

```diff
@@ -25,6 +25,8 @@ services:
   - redis-server
 
 install:
+  - go get github.com/TykTechnologies/dq
+  - go get github.com/TykTechnologies/tyk-cluster-framework/...
```

Comment:
SGTM, then if this gets merged before 2.4 we need to remember to pin/vendor before the release